product_docs/docs/epas/15/reference/sql_reference/02_data_types/pseudo_types.mdx

```diff
@@ -22,10 +22,10 @@ title: "Pseudo-types"
 
 ## Overview
 
-The PostgreSQL type system contains a number of special-purpose entries that are collectively called pseudo-types. A pseudo-type cannot be used as a column data type, but it can be used to declare a function's argument or result type. Each of the available pseudo-types is useful in situations where a function's behavior does not correspond to simply taking or returning a value of a specific SQL data type. 
+The PostgreSQL type system contains a number of special-purpose entries that are collectively called *pseudo-types*. A pseudo-type can't be used as a column data type, but it can be used to declare a function's argument or result type. Each of the available pseudo-types is useful in situations where a function's behavior doesn't correspond to taking or returning a value of a specific SQL data type. 
 
-Functions coded in C (whether built-in or dynamically loaded) can be declared to accept or return any of these pseudo data types. It is up to the function author to ensure that the function will behave safely when a pseudo-type is used as an argument type.
+Functions coded in C (whether built-in or dynamically loaded) can be declared to accept or return any of these pseudo data types. It's up to the function author to ensure that the function behaves safely when a pseudo-type is used as an argument type.
 
-Functions coded in procedural languages can use pseudo-types only as allowed by their implementation languages. At present most procedural languages forbid use of a pseudo-type as an argument type, and allow only void and record as a result type (plus trigger or event_trigger when the function is used as a trigger or event trigger). Some also support polymorphic functions using the types `anyelement`, `anyarray`, `anynonarray`, `anyenum`, and `anyrange`.
+Functions coded in procedural languages can use pseudo-types only as allowed by their implementation languages. At present, most procedural languages forbid the use of a pseudo-type as an argument type. They allow only `void` and `record` as a result type (`plus trigger` or `event_trigger` when the function is used as a trigger or event trigger). Some also support polymorphic functions using the types `anyelement`, `anyarray`, `anynonarray`, `anyenum`, and `anyrange`.
```

Comment:
Should this be (plus `trigger` or `event_trigger`) ? Not sure about the accuracy of what you've written.

Reply:
Probably the first. I'll fix it.

Reply:
```suggestion
Functions coded in procedural languages can use pseudo-types only as allowed by their implementation languages. At present, most procedural languages forbid the use of a pseudo-type as an argument type. They allow only `void` and `record` as a result type (plus `trigger` or `event_trigger` when the function is used as a trigger or event trigger). Some also support polymorphic functions using the types `anyelement`, `anyarray`, `anynonarray`, `anyenum`, and `anyrange`.
```